src/reconcile/rec_write.c

```diff
@@ -2206,9 +2206,19 @@ __wti_rec_pack_delta_internal(
 
     __wti_rec_kv_copy(session, p, key);
     p += key->len;
-    if (value == NULL)
-        LF_SET(WT_DELTA_INT_IS_DELETE);
-    else
+
+    /*
+     * If the value is NULL then write the zeroed out values and set the cell type to
+     * WT_CELL_ADDR_DEL.
+     */
+    if (value == NULL) {
+        printf("HELLO\n");
```

Comment:
remove print before merge?

Reply:
Fixed in f334c4666fc996d956f8c53444edc0a913e58079